resources/nginx.conf

```diff
@@ -0,0 +1,63 @@
+server {
+    listen       80;
+    listen  [::]:80;
+    server_name  localhost;
+
+    #access_log  /var/log/nginx/host.access.log  main;
+
+    location / {
+        root   /usr/share/nginx/html;
+        index  index.html index.htm;
+    }
+
+    # Reverse proxy redirects
+
+    location ~ ^/api\.keyman\.com\.local/?(.*) {
+        return http://127.0.0.1:8098/$1;
+    }
+
+    location ~ ^/help\.keyman\.com\.local/?(.*) {
+        return http://127.0.0.1:8055/$1;
+    }
+
+    location ~ ^/keyman\.com\.local/?(.*) {
+        return http://127.0.0.1:8053/$1;
```

Comment:
Alas, this has the effect of redirecting, so the browser URL doesn't stay http://localhost/keyman.com.local.

I tried to use `rewrite` instead, but wasn't successful...

Reply:
I inserted ll.15-29. The rest of the file was default from the nginx image.

Reply:
I've never used nginx before and haven't tried this, but something like this seems closer to what we want (so we can visit http[s]://help.keyman-local.com/products/android/...)

```
server {
  listen 80;
  server_name help.keyman-local.com help.keyman.com.local help.keyman.com.localhost;
  location / {
    proxy_pass http://127.0.0.1:8055;
  }
}

server {
  listen 80;
  server_name api.keyman-local.com api.keyman.com.local api.keyman.com.localhost;
  location / {
    proxy_pass http://127.0.0.1:8098;
  }
}

...

```

And then delete the rest of the file I think.

https://docs.nginx.com/nginx/admin-guide/web-server/reverse-proxy/